src/lib/tabs/Tabs.tsx

```diff
@@ -27,8 +27,31 @@ function a11yProps(index: any) {
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
+  appBar: {
+    backgroundColor: 'rgba(0,0,0,0)',
+    boxShadow: 'none'
+  },
   root: {
-    flexGrow: 1
+    borderRadius: '4px',
+    boxShadow: '0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)',
```

Comment:
I like the way the card and shadows look, but it's probably too opinionated for the component. Ideally we will be able to theme this component for Nybll's menu which doesn't have a border or any box-shadows. So we could probably just have this as a flat component and whoever is implementing the component can put it into an elevated card if they want to.

Reply:
This is resolved!